src/controllers/hid/hidcontroller.h

```diff
@@ -50,6 +50,8 @@ class HidController final : public Controller {
 
   protected:
     Q_INVOKABLE void send(QList<int> data, unsigned int length, unsigned int reportID = 0);
+    Q_INVOKABLE void send_feature_report(
+            QList<int> data, unsigned int length, unsigned int reportID = 0);
```

Comment:
1. This is specific to HidController, so it should be private.
2. Rename to `sendFeatureReport` for consistency
3. Remove useless `length` parameter.
4. I think this can accept a QByteArray as input so I don't think there's a need to separate this into 2 functions.

Reply:
I'm not sure if QtScript can handle QByteArrays since there are no ArrayBuffers/TypedArrays in the ES3-like environment.

Reply:
This should be merged to master so that doesn't matter.

Reply:
1.) I tried to make the method private, but then it was no longer visible in the controller mapping
2.) Renamed the function as suggested
3.) Removed the useless length parameter
4.) I removed the second function, but the call from the controller script works only with QList<int> not with QByteArray. Therefore the remaining function has the QList<int> argument.

Reply:
> This should be merged to master so that doesn't matter.

Right, then I'd argue we should even use the more general API spec where we mandate the reportID to be included in the buffer itself instead of being passed separately just of it to be prepended in the C++ later.